.gitignore

```diff
@@ -16,6 +16,10 @@
 # Mono auto generated files
```

Comment:
please check that files from obj, bin and idea folders will be removed from repository.

Reply:
I'm not sure if this has effect until not in main branch.